docs/policies/recognizing_contributors_sponsors_and_partners.rst

```diff
@@ -0,0 +1,113 @@
+Recognizing contributors, sponsors and partners
+###############################################
+
+Background information 
+======================
+
+.. vale off
+
+The Mautic Community wishes to encourage initiative/team/working group teams to acknowledge and thank their sponsors and contributors,
+but without the \*.mautic.org web properties or the Mautic application 
+becoming overly cluttered with advertising or promotional messages.
+
+Additionally, the community ensures that companies don't ‘game’ the system 
+by providing minimal contributions to be recognized (e.g. with logo placement/links).
+
+.. vale off
+
+Recognizing that the community norms around such messages are continually 
+changing and that it's probably impossible to codify a process precisely, 
+the recommendation is that we establish a policy framework for the Community 
+Leadership Team to use when evaluating each situation, and leave the final 
+decisions in the case of any disputes to the Community Council. 
+
+Definitions
+-----------
+
+vale off
+**Contributor**: there are many way to contribute to Mautic projects ranging from 
+committing code to writing good bug reports to helping with documentation to 
+doing community outreach to funding developers to work on the project. Even 
+organizations and individuals that are "merely" adopting Mautic can be extremely useful 
+contributors if they provide good, detailed, and timely feedback about requirements and bugs.
+
+**Individual Contributor**: a contributor who is a person.
+
+**Organization Contributor**: an organisation that employs a one or more people who are contributors.
+
+**Significant Contribution**: a significant contribution directly impacts the outcome of the project, not just its size.. Typically larger 
+contributions are more significant, but that doesn't preclude an extremely valuable
```

Comment:
This seems like an incomplete sentence - the original text is:

> Typically larger contributions are more significant, but that does not preclude an extremely valuable two line fix to a critical, but difficult to find, bug from being "significant". Nor does it preclude a giant code dump of rarely used code from being "not significant". As a general rule of thumb, somewhere between 25% and 50% of the contributions to a project are significant.

As policies are extremely important only changed via debate and public consultation in the Council/General Assembly, please take great care to ensure that words are an exact match.